static/js/zamboni/themes_review_templates.js

```diff
@@ -43,16 +43,18 @@ return __p;
 
 /*
 _.template(`
-<script type="text/template" id="queue-search-row-template">
   <tr class="addon-row">
     <td class="app-name">
       <span class="addon-id"><%= id %></span>
       <a href="<%= review_url %>"><%- name %></a>
     </td>
-    <td><a href="<%= reviewer %>"><%= reviewer %></a></td>
+    <td>
+      <% if (typeof reviewer !== "undefined") { %>
+        <a href="<%= reviewer %>"><%= reviewer %></a>
```

Comment:
Ok I managed to find something. The reviewer looks to be an email address. The link is totally borked and doesn't work. 

Also the data is broken somehow because the reviewer's email address is against the wrong result :/